src/stylesheets/core/mixins/_sidenav.scss

```diff
@@ -13,65 +13,67 @@
 
   a {
     border: none;
-    color: $color-base;
+    color: color('base-dark');
     display: block;
-    font-family: $font-sans;
-    line-height: 1.3;
-    padding: 0.85rem 1rem 0.85rem $site-margins-mobile;
+    font-family: font-family('body');
+    line-height: line-height($theme-font-family-base, 3);
+    padding: units(1) units($theme-site-margins-mobile);
     text-decoration: none;
 
     &:hover {
-      background-color: $color-gray-lightest;
-      color: $color-primary;
+      background-color: color('base-lightest');
+      color: color('primary');
       text-decoration: none;
     }
 
     &:focus {
       outline-offset: 0;
       position: relative;
-      z-index: 1;
+      z-index: z-index(100);
     }
 
     &.usa-current { /* stylelint-disable-line selector-no-qualifying-type */
-      border-left: $sidenav-current-border-width solid $color-primary;
-      color: $color-primary;
-      font-weight: $font-bold;
-      padding-left: $site-margins-mobile - $sidenav-current-border-width;
+      border-left: $sidenav-current-border-width solid color('primary');
+      color: color('primary');
+      font-weight: font-weight('bold');
+      padding-left: units($theme-site-margins-mobile) - $sidenav-current-border-width;
     }
   }
 }
 
 @mixin usa-sidenav-sublist {
+  $sidenav-child-padding-left: units($theme-site-margins-mobile) + units(1.5);
+  $sidenav-grandchild-padding-left: units($theme-site-margins-mobile) + units(1.5) * 2;
   @include unstyled-list();
   margin: 0;
   width: 100%;
 
   li {
     border: none;
-    font-size: $h5-font-size;
+    font-size: scale('body', 3);
   }
 
   a {
-    padding-left: 2.8rem;
-    line-height: $heading-line-height;
+    padding-left: $sidenav-child-padding-left;
+    line-height: line-height($theme-font-family-base, 3);
 
     &:hover,
     &.usa-current { /* stylelint-disable-line selector-no-qualifying-type */
```

Comment:
Does this need the `a` specificity to work properly?

Reply:
Removed in e5047d1. But I think it was just easier to write it this way.